en/components/treegrid/tree_grid.md

```diff
@@ -236,6 +236,38 @@ The indentation of the **tree cells** persists across other tree grid features l
 
 <div class="divider--half"></div>
 
+### Keyboard navigation
+
+Keyboard navigation is available by default in the Tree Grid and aims at covering as many as possible features and scenarios for the end user. When you focus a specific cell and press one of the following key combinations, the described behaviour is performed:
+
+ - `Arrow Up` - navigates one cell up (no wrapping)
```

Comment:
May be we should be using semicolons or dots at the end of all rows about key combinations